pkgs/by-name/ot/otree/package.nix

```
@@ -0,0 +1,41 @@
+{ lib
+, fetchFromGitHub
+, rustPlatform
+, stdenv
+, git
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "otree";
+  version = "0.1.0";
+
+  src = fetchFromGitHub {
+    owner = "fioncat";
+    repo = "otree";
+    rev = "v${version}";
+    hash = "sha256-ukKlypGY0+xsci/loAnf0Sz5K+82gVzkb+KnZ139cIA=";
+    leaveDotGit = true; # needed by build.rs
+  };
+
+  nativeBuildInputs = [ git ];
+
```

This requires `darwin` being declared as an input at the top:
```suggestion

  buildInputs = lib.optional stdenv.isDarwin darwin.apple_sdks.frameworks.IOKit;
```